packages/databyss-data/pouchdb/entries/lib/setBlockRelations.ts

```diff
@@ -1,44 +1,69 @@
-import { BlockRelationPayload } from '@databyss-org/editor/interfaces'
```

Comment:
Refactor so
- BlockRelationResponse is just BlockRelation
- We don't need BlockRelationOperation. we should have 2 functions: `addPageToBlockRelation(pageId: string, blockId: string, blockType: BlockType)` and `removePageFromBlockRelation(pageId: string, blockId: string)`
- We don't need BlockRelationPayload. Before we started using Typescript, we were passing objects as args to functions if they had more than one argument so that the code was more self-documenting. With Typescript, we always know which arg is which, so we don't need to do that any more. EXCEPT if there's more than one optional argument. I've been using the following pattern: `funcName(requiredArg1: someType, requiredArg2: someType, options: FuncNameOptions)`
- The `type` field on the relation doc should be renamed `blockType` for clarity
- We should add a `blockId` field for clarity, even though it has the same data as `_id` for now